### ISSUE TITLE
About training epochs.

### ISSUE THREAD

**Issue by hufengshuo07** [Daniil-Osokin/gccpm-look-into-person-cvpr19.pytorch#1]
I found that there isn't well-trained checkpoints, so I need train from scratch, could you tell me total number of iterations or epochs to approach your 84% results?

**Comment by Daniil-Osokin** [comment]
Hi, it converges after about 50 epochs (learning rate drops two times).

**Comment by hufengshuo07** [comment]
@Daniil-Osokin Oh, thanks. For myself, my training takes 38 hours on 1xP100 to converge with batchsize=64, pckh~84.4. Hope to help others

**Comment by Daniil-Osokin** [comment]
Thanks for the report back your results!

**Comment by qintao361** [comment]
> @Daniil-Osokin Oh, thanks. For myself, my training takes 38 hours and 70epochs on 1xP100 to converge with batchsize=64, pckh~84.4. Hope to help others

I have the same probelm for loading pre_trained model for initialization.
could you provide your pre_trained model for us? thank you so much
is it possible upload the model to Google Drive or Baidu Drive?

**Comment by qintao361** [comment]
> I found that there isn't well-trained checkpoints, so I need train from scratch, could you tell me total number of iterations or epochs to approach your 84% results?

I have the same probelm for loading pre_trained model for initialization.
could you provide your pre_trained model for us? thank you so much
is it possible upload the model to Google Drive or Baidu Drive?

**Comment by Daniil-Osokin** [comment]
Hi, @qintao361! I will try to do it next week.

**Comment by Daniil-Osokin** [comment]
Check it out: #12, 70 epochs and 84.57 PCKh.
